test/malli/core_test.cljc

```diff
@@ -3581,3 +3581,356 @@
   (is (not (m/validate [:sequential {:min 11} :int] (eduction identity (range 10)))))
   (is (not (m/validate [:seqable {:min 11} :int] (eduction identity (range 10)))))
   (is (nil? (m/explain [:sequential {:min 9} :int] (eduction identity (range 10))))))
+
+(deftest xor-test
+  (is (= [:xor [:= 1] [:= 2]]
+         (m/form (m/schema [:xor [:= 1] [:= 2]] {:registry (merge (mu/schemas) (m/default-schemas))}))))
```

Comment:
PR adds new schemas to default registry, here expectation is that they are in `malli.util`.